presto-product-tests-launcher/src/main/java/io/prestosql/tests/product/launcher/env/common/Standard.java

```diff
@@ -0,0 +1,96 @@
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.prestosql.tests.product.launcher.env.common;
+
+import io.prestosql.tests.product.launcher.PathResolver;
+import io.prestosql.tests.product.launcher.docker.DockerFiles;
+import io.prestosql.tests.product.launcher.env.DockerContainer;
+import io.prestosql.tests.product.launcher.env.Environment;
+import io.prestosql.tests.product.launcher.env.EnvironmentOptions;
+import org.testcontainers.containers.startupcheck.IsRunningStartupCheckStrategy;
+import org.testcontainers.containers.wait.strategy.Wait;
+import org.testcontainers.containers.wait.strategy.WaitAllStrategy;
+
+import javax.inject.Inject;
+
+import java.io.File;
+import java.time.Duration;
+
+import static com.google.common.base.Preconditions.checkArgument;
+import static io.prestosql.tests.product.launcher.testcontainers.TestcontainersUtil.exposePort;
+import static java.util.Objects.requireNonNull;
+import static org.testcontainers.containers.BindMode.READ_ONLY;
+
+public final class Standard
+        implements EnvironmentExtender
+{
+    private final PathResolver pathResolver;
+    private final DockerFiles dockerFiles;
+
+    private final String imagesVersion;
+    private final File serverPackage;
+
+    @Inject
+    public Standard(
+            PathResolver pathResolver,
+            DockerFiles dockerFiles,
+            EnvironmentOptions environmentOptions)
+    {
+        this.pathResolver = requireNonNull(pathResolver, "pathResolver is null");
+        this.dockerFiles = requireNonNull(dockerFiles, "dockerFiles is null");
+        requireNonNull(environmentOptions, "environmentOptions is null");
+        imagesVersion = requireNonNull(environmentOptions.imagesVersion, "environmentOptions.imagesVersion is null");
+        serverPackage = requireNonNull(environmentOptions.serverPackage, "environmentOptions.serverPackage is null");
+        checkArgument(serverPackage.getName().endsWith(".tar.gz"), "Currently only server .tar.gz package is supported");
+    }
+
+    @Override
+    public void extendEnvironment(Environment.Builder builder)
+    {
+        builder.addContainer("presto-master", createPrestoMaster());
+        builder.addContainer("tests", createTestsContainer());
+    }
+
+    @SuppressWarnings("resource")
+    private DockerContainer createPrestoMaster()
+    {
+        DockerContainer container = new DockerContainer("prestodev/centos7-oj11:" + imagesVersion)
+                .withFileSystemBind(dockerFiles.getDockerFilesHostPath(), "/docker/presto-product-tests", READ_ONLY)
+                .withFileSystemBind(pathResolver.resolvePlaceholders(serverPackage).toString(), "/docker/presto-server.tar.gz", READ_ONLY)
+                .withFileSystemBind(dockerFiles.getDockerFilesHostPath("common/standard/config.properties"), "/docker/presto-product-tests/conf/presto/etc/config.properties", READ_ONLY)
```

Comment:
> Also I would move common/standard/config.properties to conf/environment/singlenode/config.properties. It seems to be common, but

I can't mount a config file that doens't exist, because I am mounting a config volume RO.
For good or bad. Bad -- i need default config for presto and for every catalog that we use.
Good -- we won't accidentally create zero-length empty files that cause subsequent failures.

> Please extract constant for "/docker/presto-product-tests/conf/presto/etc/config.properties" as this is going to be a path for all config properties for all presto environments.

Good idea